gtsam.h

```diff
@@ -120,6 +120,12 @@
 
 namespace gtsam {
 
+// Include Key typedef
+#include <gtsam/base/types.h>
+class Key {
```

Comment:
Let's not do that in this PR - I think it's actually not a good idea. I'll explain why in person